templates/helm/values.yaml.tpl

```diff
@@ -85,3 +85,7 @@ serviceAccount:
   name: {{ .ServiceAccountName }}
   annotations: {}
     # eks.amazonaws.com/role-arn: arn:aws:iam::AWS_ACCOUNT_ID:role/IAM_ROLE_NAME
+
+driftRemediation:
```

Comment:
How about we call this configuration section `reconcile`?

Reply:
++